src/spring-bceid-starter/src/main/java/ca/bc/gov/open/bceid/starter/account/mappers/NameMapper.java

```diff
@@ -0,0 +1,18 @@
+package ca.bc.gov.open.bceid.starter.account.mappers;
+
+import ca.bc.gov.open.bceid.starter.account.models.Name;
+import ca.bceid.webservices.client.v9.BCeIDName;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(uses = BCeIdMapper.class)
+public interface NameMapper {
+
+    @Mapping(target = "firstName", source = "firstname", qualifiedByName=  { "BCeIdMapper", "toString"} )
+    @Mapping(target = "middleName", source = "middleName", qualifiedByName=  { "BCeIdMapper", "toString"} )
+    @Mapping(target = "otherMiddleName", source = "otherMiddleName", qualifiedByName=  { "BCeIdMapper", "toString"} )
+    @Mapping(target = "surname", source = "surname", qualifiedByName=  { "BCeIdMapper", "toString"} )
```

Comment:
interesting, I thought it was called `lastName` not `surname`

Reply:
that's when it's going out to BCeId, it's called surname....